_posts/2020-04-13-pins-04/pins-04.Rmd

```diff
@@ -0,0 +1,174 @@
+---
+title: "pins 0.4: Versioning"
+description: > 
+  A new release of pins is available on CRAN today. This release adds support to time travel across dataset versions, which improves collaboration and protects your code from breaking when remote resources change unexpectedly.
+author:
+  - name: Javier Luraschi
+    affiliation: RStudio
+    affiliation_url: https://twitter.com/javierluraschi
+slug: pins-0.4
+date: 04-13-2020
+categories:
+  - Packages/Releases
+  - TensorFlow/Keras
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.jpg
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+A new version of `pins` is available on CRAN today, which adds support for [versioning](http://pins.rstudio.com/articles/advanced-versions.html) your datasets and [DigitalOcean Spaces](http://pins.rstudio.com/articles/boards-dospace.html) boards!
+
+As a quick recap, the pins package allows you to cache, discover and share resources. You can use `pins` in a wide range of situations, from downloading a dataset from a URL, to creating complex automation workflows, learn more about at [pins.rstudio.com](https://pins.rstudio.com). You can also use `pins` in combination with TensorFlow and Keras; for instance, you can use the [cloudml](https://tensorflow.rstudio.com/tools/cloudml) to train models in cloud GPUs, but rather than manually copying files into the GPU instance, you can use pins directly from R.
```

Comment:
```suggestion
As a quick recap, the pins package allows you to cache, discover and share resources. You can use `pins` in a wide range of situations, from downloading a dataset from a URL to creating complex automation workflows (learn more at [pins.rstudio.com](https://pins.rstudio.com). You can also use `pins` in combination with TensorFlow and Keras; for instance, you can use the [cloudml](https://tensorflow.rstudio.com/tools/cloudml) to train models in cloud GPUs, but rather than manually copying files into the GPU instance, you can use pins directly from R.
```